From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 8 Apr 2020 16:24:23 +0200
Subject: units: drop systemd-remount-fs.service dependency from more services

All services using StateDirectory= don't need the explicit dep anymore,
let's hence drop it everywhere.

(cherry picked from commit e0f968ad96e9a6340edced8e22089c802c2f6497)
---
 units/systemd-backlight@.service.in | 1 -
 units/systemd-coredump@.service.in  | 2 +-
 units/systemd-rfkill.service.in     | 2 +-
 units/systemd-timesyncd.service.in  | 2 +-
 4 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/units/systemd-backlight@.service.in b/units/systemd-backlight@.service.in
index d302285..6f7967f 100644
--- a/units/systemd-backlight@.service.in
+++ b/units/systemd-backlight@.service.in
@@ -12,7 +12,6 @@ Description=Load/Save Screen Backlight Brightness of %i
 Documentation=man:systemd-backlight@.service(8)
 DefaultDependencies=no
 Conflicts=shutdown.target
-After=systemd-remount-fs.service
 Before=sysinit.target shutdown.target
 
 [Service]
diff --git a/units/systemd-coredump@.service.in b/units/systemd-coredump@.service.in
index 951faa6..d1ece27 100644
--- a/units/systemd-coredump@.service.in
+++ b/units/systemd-coredump@.service.in
@@ -12,7 +12,7 @@ Description=Process Core Dump
 Documentation=man:systemd-coredump(8)
 DefaultDependencies=no
 Conflicts=shutdown.target
-After=systemd-remount-fs.service systemd-journald.socket
+After=systemd-journald.socket
 Requires=systemd-journald.socket
 Before=shutdown.target
 
diff --git a/units/systemd-rfkill.service.in b/units/systemd-rfkill.service.in
index 3abb958..af2042f 100644
--- a/units/systemd-rfkill.service.in
+++ b/units/systemd-rfkill.service.in
@@ -13,7 +13,7 @@ Documentation=man:systemd-rfkill.service(8)
 DefaultDependencies=no
 BindsTo=sys-devices-virtual-misc-rfkill.device
 Conflicts=shutdown.target
-After=sys-devices-virtual-misc-rfkill.device systemd-remount-fs.service
+After=sys-devices-virtual-misc-rfkill.device
 Before=shutdown.target
 
 [Service]
diff --git a/units/systemd-timesyncd.service.in b/units/systemd-timesyncd.service.in
index f0486a7..92ee945 100644
--- a/units/systemd-timesyncd.service.in
+++ b/units/systemd-timesyncd.service.in
@@ -13,7 +13,7 @@ Documentation=man:systemd-timesyncd.service(8)
 ConditionCapability=CAP_SYS_TIME
 ConditionVirtualization=!container
 DefaultDependencies=no
-After=systemd-remount-fs.service systemd-sysusers.service
+After=systemd-sysusers.service
 Before=time-set.target sysinit.target shutdown.target
 Conflicts=shutdown.target
 Wants=time-set.target time-sync.target
